tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -1987,12 +1985,8 @@ public function theAdministratorHasAddedTheFollowingUsersToAGroupAtOnceUsingTheG
 		foreach ($table->getHash() as $row) {
 			$userIds[] = $this->featureContext->getAttributeOfCreatedUser($row['username'], "id");
 		}
-		$this->addMultipleUsersToGroup($user, $userIds, $groupId, $table);
-		$response = $this->featureContext->getResponse();
-		if ($response->getStatusCode() !== 204) {
-			$this->throwHttpException($response, "Cannot add users to group '$group'");
-		}
-		$this->featureContext->emptyLastHTTPStatusCodesArray();
+		$response = $this->addMultipleUsersToGroup($user, $userIds, $groupId, $table);
```

Comment:
I didn't see use for $table in `addMultipleUsersToGroup`. column verification can/should be done within stef def (wherever possible)
```suggestion
		$response = $this->addMultipleUsersToGroup($user, $userIds, $groupId);
```